app/services/katello/candlepin/consumer.rb

```diff
@@ -1,8 +1,16 @@
 module Katello
   module Candlepin
     class Consumer
+      include LazyAccessor
       attr_accessor :uuid
 
+      lazy_accessor :entitlements, :initializer => lambda { |_s| Resources::Candlepin::Consumer.entitlements(uuid) }
+      lazy_accessor :events, :initializer => lambda { |_s| Resources::Candlepin::Consumer.events(uuid) }
+      lazy_accessor :consumer_attributes, :initializer => lambda { |_s| Resources::Candlepin::Consumer.get(uuid) }
+      lazy_accessor :installed_products, :initializer => lambda { |_s| consumer_attributes['installedProducts'] }
+      lazy_accessor :available_pools, :initializer => lambda { |_s| Resources::Candlepin::Consumer.available_pools(uuid, false) }
+      lazy_accessor :all_available_pools, :initializer => lambda { |_s| Resources::Candlepin::Consumer.available_pools(uuid, true) }
```

Comment:
putting this above line 5 so the attr_accessor is near the initialize method would make this file clearer to parse for me.